api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -452,6 +453,13 @@ message RemoteJwks {
   //
   //
   config.core.v3.RetryPolicy retry_policy = 4;
+
+  // Refetch JWKS if extracted JWT has no KID or a KID that does not match cached JWKS's KID.
+  // Async JWKS fetching is independent to this refetching process. New JWKS is fetched on
```

Comment:
What does this sentence mean?

Reply:
There is an async JWKS fetching process that would happen at regular intervals, if configured by end user: https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/jwt_authn/v3/config.proto.html#envoy-v3-api-msg-extensions-filters-http-jwt-authn-v3-jwksasyncfetch. The async fetching process won't interfere with this refetching process and vice-versa. Happy to reword to add more clarify, if required.

Reply:
Got it, thanks.  I suggest referencing the `async_fetch` field here, so it's clear what this is referring to.